ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -268,6 +268,21 @@ StackLayout {
                 onStickerSelected: {
                     chatsModel.sendSticker(hashId, packId)
                 }
+                onSendMessage: {
+                    if (chatInput.fileUrls.length > 0){
+                        chatsModel.sendImage(chatInput.fileUrls[0]);
+                    }
+                    var msg = chatsModel.plainText(Emoji.deparse(chatInput.textInput.text).trim()).trim()
```

Comment:
FYI, this line was changed by Hydrogen. He removed the `trim`s since they caused issues in code blocks

Reply:
uhm.. I believe the change isn't there yet, otherwise there'd be a conflict

Reply:
Indeed, you can see it here: https://github.com/status-im/nim-status-client/pull/1475